index.html

```diff
@@ -2690,8 +2689,10 @@
           Exposing user information
         </h2>
         <p>
-          The <a>user agent</a> MUST NOT share information about the user to
-          the web page (such as the shipping address) without user consent.
+          A page might try to call the payment request API to retrieve
+          information about the user. Information about the user (such as the
+          shipping address) should only be shared with the web page with user
```

Comment:
I think it's crazy to try to make this say should without using the word should and this is a pretty important recommendation that has come up over and over in the group discussion but be my guest if you want to update this with something else.